packages/loader/container-loader/src/quorum.ts

```diff
@@ -32,3 +34,26 @@ export class QuorumProxy extends EventForwarder<IQuorumEvents> implements IQuoru
         this.getMember = doIfNotDisposed(this, quorum.getMember.bind(quorum));
     }
 }
+
+export function getCodeProposalFromQuorumValues(
```

Comment:
I got a little confused by the terms "code proposal" being used for both `IFluidCodeDetails` and `ICommittedProposal`. Maybe it will help to rename this to `getCodeDetailsFromQuorumValues` and use the variable names `codeDetails` for `IFluidCodeDetails` everywhere?